### ISSUE TITLE
When booting into xCAT genesis shell, the status should be updated to be consistent

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#4248]
Looking at this node: 
```
[root@briggs01 .ssh]# lsdef mid05tor12cn11 -i provmethod,status
Object name: mid05tor12cn11
    currchain=boot
    currstate=shell
    provmethod=rhels7.4-alternate-ppc64le-install-compute
    status=powering-on
```

There's no update on the status here,  we should update something to get it into a recognized state, perhaps it should be `status=shell`   Because leaving it in powering-on seems to be an error....

**Comment by robin2008** [comment]
Agree, and I think it might be a long story for us to get a clear state transition.

we have some discussion when implementing 'xcatprobe clusterstatus' feature, and plan to enhance the state transition for different kinds of operation if we hit.

**Comment by immarvin** [comment]
agree that the status update should be enhanced for procedures besides provision

**Comment by whowutwut** [comment]
I was talking to @cxhong today and it seems like we should have some system status (powering-on, netbooting/installing,booted) and then it should go into system status where we run the post script and after all xcat postscript processing is done. Maybe end up with ready state

I know some folks in the past have asked how do we know when all of xcat postscript is complete. It might be good to update some file on the machine to contain the status so applications can check this "ready" or final state. Just some ideas. Maybe not the right issue to discuss in

**Comment by zet809** [comment]
Seems related to issue #3902

**Comment by xuweibj** [comment]
Debug record:

1. Add change status module for shell/standby state in "setdestiny" function in destiny.pm.

Scenario 1:
2. Run ``nodeset shell/standby`` against CN.
3. Check status of CN, it's "shell/standingby"
4. To let CN enter shell/standingby, run ``rpower boot``.
5. Check status, is "powering-on"

Scenario 2:
2. Run ``nodeset shell,standby`` against CN. It's same with the scenario 1 until exit shell.
3. After exit shell, status is changed to "standingby"

To do:
1. CN send status when enter shell/standby.
2. MN should receive it and change status.
